common/main/bg.xml

```diff
@@ -305,6 +305,7 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<language type="li">лимбургски</language>
 			<language type="lil">лилоует</language>
 			<language type="lkt">лакота</language>
+			<language type="lmo" draft="contributed">ломбардски</language>
```

Comment:
Hmm, added as in Arabic. Were we collecting names for this?

Reply:
`lmo` is a new locale for basic. did it get special handling?

Reply:
Missing values (needed for CLDR target coverage) could be added. If we added a new language type, then that created a new Missing item. If that was done at the end of submission, then like the beaufort paths, we need to subtract from coverage (AND not do it in the future!)

Reply:
Investigated in https://unicode-org.atlassian.net/browse/CLDR-16347  - it was due to being missing.

Reply:
Same with Tatar